BookClub/tests/views/test_delete_booklist_view.py

```diff
@@ -0,0 +1,111 @@
+from django.test import TestCase, tag
+from django.urls import reverse
+from django.contrib import messages
+
+from BookClub.models import User, Book, BookList
+from BookClub.tests.helpers import LogInTester, reverse_with_next
+
+@tag('booklist', 'book', 'user')
+@tag('booklist_view', 'deletebooklist')
+class DeleteBookListView(TestCase, LogInTester):
+
+    fixtures = [
+        'BookClub/tests/fixtures/default_users.json',
+        'BookClub/tests/fixtures/default_books.json',
+        'BookClub/tests/fixtures/booklists.json'
+    ]
+
+    def setUp(self):
+        self.user = User.objects.get(username = 'johndoe')
+        self.booklist = BookList.objects.get(pk = 1)
+        self.url = reverse('delete_booklist', kwargs = {'username' : self.user.username,
+                                                        'list_id' : self.booklist.pk})
+
+    def test_url(self):
+        correct_url = '/user/johndoe/list/1/delete/'
+        self.assertEqual(self.url, correct_url)
+
+    def test_redirects_on_get_request_when_not_logged_in(self):
+        self.assertFalse(self._is_logged_in())
+
+        redirect_url = reverse_with_next('login', self.url)
+        response = self.client.get(self.url, follow = True)
+        self.assertRedirects(response, redirect_url,
+            status_code=302, target_status_code=200, fetch_redirect_response=True
+        )
+        self.assertTemplateUsed(response, 'login.html')
+
+    def test_redirects_on_get_request_when_logged_in(self):
+        self.client.login(username='johndoe', password='Password123')
+        self.assertTrue(self._is_logged_in())
+
+        redirect_url = reverse('booklists_list', kwargs={'username' : 'johndoe'})
+        response = self.client.get(self.url, follow = True)
+        self.assertRedirects(response, redirect_url,
+            status_code=302, target_status_code=200, fetch_redirect_response=True
+        )
+        self.assertTemplateUsed(response, 'user_booklists.html')
+
+        response_messages = messages.get_messages(response.wsgi_request)
+        self.assertEqual(len(response_messages), 1)
+        # below test not working, cannot check message contents...
```

Comment:
try something like: 
messages_list = list(response_message.context['messages'])
        self.assertEqual(len(messages_list),1)
        self.assertEqual(str(messages_list[0]) ,'Messages contents here')
        self.assertEqual(messages_list[0].level,messages.ERROR/SUCCESS)

Reply:
Fixed now in latest commit.